### ISSUE TITLE
Per bucket validations

### ISSUE THREAD

**Issue by knownasilya** [offirgolan/ember-state-manager#1]
Would love an example of validations with a buffer on a per bucket basis. How would one do that?

**Comment by knownasilya** [comment]
Ah, I think http://offirgolan.github.io/ember-cp-validations/docs/modules/Advanced.html#nested-keys is the answer.

**Comment by offirgolan** [comment]
Do you mean that each bucket should be able to have a different set of validations?

**Comment by knownasilya** [comment]
Yeah, at least that's how I envisioned buckets when I saw them documented. I did realize that bucket just means a clone of the initial state, but it would be interesting to have groups of state. So `states/user.js` would allow for a user model, but also would have other "buckets" relating to the user.
